README.md

```diff
@@ -1,78 +1,26 @@
 # 👩🏻‍🍳 Haystack Cookbook
 
 <div align="center">
-  <a href="https://haystack.deepset.ai/"><img src="https://github.com/deepset-ai/haystack/blob/main/docs/img/banner.png" alt="Green logo of a stylized white 'H' with the text 'Haystack, by deepset. Haystack 2.0 is live 🎉' Abstract green and yellow diagrams in the background."></a>
+  <a href="https://haystack.deepset.ai/"><img src="https://github.com/deepset-ai/haystack/blob/main/docs/img/banner.png" alt="Green logo of a stylized white 'H' with the text 'Haystack, by deepset. The Production-Ready Open Source AI Framework' Abstract green and yellow diagrams in the background."></a>
 </div>
 
-# 🧑‍🍳🍳Discover The Haystack Cookbook [here](https://haystack.deepset.ai/cookbook)
+## 🧑‍🍳🍳Discover The Haystack Cookbook [here](https://haystack.deepset.ai/cookbook)
 
-A collection of example notebooks using [Haystack](https://github.com/deepset-ai/haystack) 👇
+A collection of example notebooks using [Haystack](https://github.com/deepset-ai/haystack) 💚
 
-You can use these examples as guidelines on how to make use of different model providers, vector databases, retrieval techniques and more with Haystack. Most of them showcase a specific, small demo.
+These examples can serve as guidelines for using different model providers, vector databases, retrieval techniques, new experimental features, and more with Haystack. Most showcase a specific, small demo.
 
-> 🧑‍🍳 [Guidelines on How to Contribute a Cookbook](#how-to-contribute-to-this-repository)
+> 🧑‍🍳 [Guidelines on How to Contribute a Cookbook](#how-to-contribute-a-cookbook)
```

Comment:
It's more like contributing a recipe to the cookbook instead of contributing a book right? :D 
So maybe `Guidelines on How to Contribute to the Cookbook`?